src/ipc/Queue.h

```diff
@@ -15,9 +15,14 @@
 #include "ipc/mem/Pointer.h"
 #include "util.h"
 
+#include <algorithm>
 #include <atomic>
 
 class String;
+class StoreEntry;
+
+// Branch XXX: storeAppendPrintf is deprecated in favor of StoreEntry::appendf()
+void storeAppendPrintf(StoreEntry *, const char *,...) PRINTF_FORMAT_ARG2;
```

Comment:
I thought we already fixed similar problems, and now I accidentally found that existing solution: If possible, please use PackableStream instead of printf(). PackableStream will also address the remaining problems with mismatching `%u` codes (another branch XXX).

Reply:
Done at 5d22445 and abcb0f3.